app/templates/index.html

```diff
@@ -142,6 +142,7 @@ <h3 class="header center">Translation API</h3>
 						<div class="row mb-0">
 							<div class="col s6 language-select">
 								<span>Translate from</span>
+                <span><i>[[ detectedLangText ]]</i></span>
```

Comment:
I think it should rather display the language and not the ISO code

Reply:
done in https://github.com/LibreTranslate/LibreTranslate/pull/324

Reply:
Perfect! Was working on this improvement too at the moment! Thank you! :)